integration-tests/deployment/ccip/state.go

```diff
@@ -49,6 +54,63 @@ type CCIPChainState struct {
 	TestRouter *router.Router
 }
 
+func (c CCIPChainState) GenerateView() (view.ChainView, error) {
+	chainView := view.NewChain()
+	r := c.Router
```

Comment:
ultranit - we can avoid these variables and just use c.Router then if c.Router != nil {c.Router.Address()}